### PR TITLE
Adds LIBMAX AND MAXTUL FASL files.  

### LINKED ISSUE #710: Build LIBMAX Lisp FASL files to support Macsyma
Macsyma relies on several libraries from LIBLSP and quite a few libraries from LIBMAX. This ticket, which is a prerequisite for a successful Macsyma build, calls for building all the relevant LIBMAX FASL files that are required for Macsyma. There are some additional LIBMAX files not covered by this ticket, but these will be addressed in a later ticket.  The LIBMAX FASL files required for Macyma to build are: MODULE, MAXMAC, MFORMA, LUSETS, ERMSGX, ERMSGC, LMMAC, META, LMRUND, LMRUN, DISPLM, DEFOPT, MOPERS, MRGMAC, NUMMAC, OPSHIN, EDMAC, PROCS, READM, STRMAC, TRANSM, RZMAC, TRANSQ, MDEFUN, and DEFINE.  

A subsequent ticket will augment this list. MHAYAT and RATMAC cannot be compiled until additional supporting FASL files are available and compiled.

### PR DESCRIPTION
**Pull request by eswenson1**
These are prerequisites for building and running Macsyma.  Resolves #710 and #711.

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
I'll look into the KA10 failure.

**Comment by eswenson1** [comment]
I imagine it is likely caused by the significantly longer/larger dump that results from all the new files.

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ SRC = system syseng sysen1 sysen2 sysen3 sysnet kshack dragon channa	\
       midas _teco_ emacs emacs1 rms klh syshst sra mrc ksc eak gren	\
       bawden _mail_ l lisp libdoc comlap lspsrc nilcom rwk chprog rg	\
       inquir acount gz sys decsys ecc alan sail kcc kcc_sy c games archy dcp \
-      spcwar rwg
+      spcwar rwg libmax rwk rat z emaxim rz maxtul
```
Comment (larsbrinkhoff) [inline Makefile]: `rwk` is already in the list.

**Comment by larsbrinkhoff** [comment]
I have started a build on my own machine.  Maybe we can disable use of async I/O in KA10.

**Comment by larsbrinkhoff** [comment]
Sorry, I don't have a solution or workaround yet.  https://github.com/rcornwell/sims/issues/53

**Comment by larsbrinkhoff** [comment]
Apparently, something tries to write a tape record longer than 32K bytes.

I don't think this should necessary block Macsyma.  Either we push this as is and accept the failed build status, or we disable KA10 for now knowing there's a bug that must be fixed.

**Comment by eswenson1** [comment]
Agreed.

**Comment by larsbrinkhoff** [comment]
Ok.  Fix the duplicated rwk, comment out sims in .travis.yml, and Bob's your uncle!

**Comment by eswenson1** [comment]
Ok, will do.

**Comment by eswenson1** [comment]
Richard just said he solved the problem. Maybe I should wait?

**Comment by larsbrinkhoff** [comment]
If you have the patience. :-)

**Comment by eswenson1** [comment]
Patience?  What's that?  :-)

**Comment by eswenson1** [comment]
I'll wait to see if Richard can fix the issue with tape writing before moving forward with this one.  He's suggested he'll test the fix tonight.

**Comment by eswenson1** [comment]
Pushed a fix for duplicate rwk source directory.  Waiting, now, to see if Travis will be able to build the SIMS version.

**Comment by eswenson1** [comment]
I don't think you, @larsbrinkhoff, have approved this PR yet.  So I won't merge just yet.  It does build now on all three simulators.

**Comment by larsbrinkhoff** [comment]
That's because I had a comment on Makefile.  That's fixed now, so I'll merge.

**Comment by larsbrinkhoff** [comment]
I found a problem in this pull request:
https://travis-ci.org/PDP-10/its/jobs/351178190#L6325

The file RLB%; FASDMP 124 was added, but rlb% was not added to the Makefile.  So it's not copied to the sources tape, and building it fails.  But the script doesn't take notice of this.

This is fixed in the follow up pull request.

**Comment by eswenson1** [comment]
Yeah, I noticed that when I built the branch locally. I didn’t know how to stop the running Travis build, so just pushed a fix to the Makefile. A new Travis build was queued, but the original one had to complete (and fail) first.

**Pull request merged**